src/core/utils/helpers/list-mapper.ts

```diff
@@ -36,6 +36,24 @@ function getYearFromDataString(date: string) {
   return new Date(date).getFullYear();
 }
 
+function getSeriesString(
+  series: {
+    title: string;
+    numberInSeries?: {
+      number?: Array<number> | null;
+    } | null;
+  }[]
+) {
+  return series
+    .map(({ title, numberInSeries }) => {
+      if (numberInSeries && numberInSeries.number) {
+        return `${title} ${numberInSeries?.number?.[0]}`;
```

Comment:
since you already checked for `numberInSeries.number` it should only be necessary to know if key 0 exists.